modules/builds-docker-credentials-private-registries.adoc

```diff
@@ -112,7 +112,7 @@ This example uses `pullSecret` in a Source build, but it is also applicable in D
 ====
 endif::[]
 +
-You can also link the pull secret to the service account used by the build instead of specifying the `pullSecret` field. By default, builds use the `builder` service account. The pull secret is automatically added to the build if the secret contains a credential that matches the repository hosting the build's input image. To link the pull secret to the service account used by the build instead of specifying the `pullSecret` field, run:
+You can also link the pull secret to the service account used by the build instead of specifying the `pullSecret` field. By default, builds use the `builder` service account. The pull secret is automatically added to the build if the secret contains a credential that matches the repository hosting the build's input image. To link the pull secret to the service account used by the build instead of specifying the `pullSecret` field, enter the following command:
```

Comment:
I agree this is dense. I wonder if some of it could be pulled out into a NOTE. Maybe the middle two sentences?

Reply:
Agreed. I think I'll leave it for now. I'm actually unsure if this para is something you can do in addition to the previous command, or if it's an alternative to it. The steps in this procedure could really be improved, but I think it would be better for the writer who covers this area to handle that.